internal/venice-common/src/main/java/com/linkedin/venice/pubsub/manager/TopicManager.java

```diff
@@ -522,10 +478,15 @@ public void ensureTopicIsDeletedAndBlock(PubSubTopic pubSubTopic) {
 
     logger.info("Deleting topic: {}", pubSubTopic);
     try {
-      pubSubWriteOnlyAdminAdapter.get().deleteTopic(pubSubTopic, Duration.ofMillis(kafkaOperationTimeoutMs));
+      long startTime = System.currentTimeMillis();
```

Comment:
Having a generic question about the admin operation latency:
Will these operation take more than 1ms or less than 1ms?
If these operations would normally take less than 1ms, we need to use nano seconds here, otherwise, the latency metric can report 0.

Reply:
based on what I've observed, the latencies of pub-sub ops typically fall within the millisecond or sub-millisecond range. To obtain more accurate measurements, we can convert these latencies to microsecond.

Reply:
Update to record latency in microseconds instead of millis

Reply:
Note that all metrics are in floating point, so even if we measure in nanos, we can still report it as fractional millis (i.e. via `LatencyUtils::getLatencyInMS`). Microseconds are uncommon, however... I don't know if we have any other metrics in that unit. I suggest we stick to (fractional) millis for conformity with the rest of the metrics code.

Reply:
Updated the PR to use `LatencyUtils::getLatencyInMS`